src/components/RestaurantSidebar/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Style from './styles.module.css'
+
+const RestaurantSidebar = ({items}) => {
+  return (
+    <div className={Style.sidebar}>
+      <ul>
+        {items.map((item, index) => {
+            return (
+                <li className={Style.li} key={index}>
```

Comment:
While using the index variable as the key property is acceptable when there are no unique identifiers available, using a unique identifier such as the id property of each item is preferable for improved rendering performance and avoiding issues with reordering or deleting items.